src/API.ts

```diff
@@ -0,0 +1,47 @@
+import { useCategory } from './CategoryContext';
+import { shuffleArray } from './utils';
+
+
+
+
+
+
+export type Question = {
+  category: string;
+  correct_answer: string;
+  difficulty: string;
+  incorrect_answers: string[];
+  question: string;
+  type: string;
+};
+
+export enum Difficulty {
+  EASY = "easy",
+  MEDIUM = "medium",
+  HARD = "hard",
+}
+
+export type QuestionsState = Question & { answers: string[] };
+
+
+  // const{category}=useCategory()
+
+ export const fetchQuizQuestions = async (amount: number, difficulty: Difficulty): Promise<QuestionsState[]> => {
+
+ ;
+  // const endpoint = `https://opentdb.com/api.php?amount=${amount}&difficulty=${difficulty}&type=multiple`;
```

Comment:
remove all the commented code

Reply:
many of these commented lines will be used in next version app, so haven't removed it will remove them before final version